owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -914,7 +915,11 @@ class FileDisplayActivity : FileActivity(),
             is UIResult.Error -> {
                 dismissLoadingDialog()
 
-                showErrorInSnackbar(R.string.move_file_error, uiResult.error)
+                uiResult.error?.let {
+                    showMessageInSnackbar(
+                        message = it.parseError(getString(R.string.copy_file_error), resources, true)
```

Comment:
True, copy&paste 😬. Anyway, as the last parameter is `true` (showJustReason), that first parameter won't be ever shown. But just in case, let's pass the correct parameter 👍